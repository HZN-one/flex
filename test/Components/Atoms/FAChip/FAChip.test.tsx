/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FAChip } from "@Components";

import { fireEvent, render } from "@testing-library/react";

describe("FAChip", () => {
  it("should match snapshot", () => {
    const { container } = render(<FAChip testID="chip-example" />);
    expect(container).toMatchSnapshot();
  });
});
