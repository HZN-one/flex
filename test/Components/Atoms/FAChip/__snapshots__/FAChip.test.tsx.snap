// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAChip should match snapshot 1`] = `
<div>
  <div
    class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-1iz24ed-MuiChip-root"
    data-testid="chip-example"
  >
    <span
      class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
    />
  </div>
</div>
`;
